style/calendar.less

```diff
@@ -80,6 +80,24 @@ body.modal-open {
     .bootstrap-column-padding-override();
   }
 
+  .calendar-loading {
+    .rc-Month:after {
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      content: 'Loading Plans...';
```

Comment:
I don't think teachers know what plans are. Maybe `Loading Calendar...`? Thoughts on the loading screen @Fredasaurus (see PR Screenshot)